### ISSUE TITLE
Insufficient Contrast for review timestamps

### ISSUE THREAD

**Issue by ravenokko** [shopware/shopware#15122]
### Shopware Version

trunk

### Affected area / extension

Platform(Default)

### Actual behaviour

The date/timestamps for reviews currently have a contrast of `3.8:1` which is a WCAG violation. 

<img width="2862" height="1228" alt="Image" src="https://github.com/user-attachments/assets/39b57bd0-5081-41eb-ae4f-3e4f82bea43f" />

### Expected behaviour

Per [SC 1.4.3](https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum.html) text must have a contrast ratio of at least `4.5:1` at this size.

### How to reproduce

Open [Web Aim contrast checker](https://webaim.org/resources/contrastchecker/)
Copy current color HEX: #798490
Paste value into Foreground.
